src/templates/antora/antora.converter.ts

```diff
@@ -364,5 +368,6 @@ const convert = (pages: PageOutput[][]): PageOutput[][] => pages.map((page) => {
 });
 
 export {
+  getFilePath,
```

Comment:
This logic probably should be pulled out into a utility module or similar if we now need it in multiple places.